src/GeositeFramework/js/Export.js

```diff
@@ -41,6 +41,19 @@ require(['use!Geosite'],
                 return 'css/print-portrait.css';
             }
         }
+        
+        function canSetPageLayout() {
```

Comment:
To add, non-Chrome browsers expose an `onafterprint` event, which is what is used to destroy the modified styles and put the map back in place. The core problem is that `window.print` didn't consistently block across browsers.